dot/state/grandpa.go

```diff
@@ -228,8 +236,12 @@ func (s *GrandpaState) ApplyForcedChanges(importedBlockHeader *types.Header) err
 
 	logger.Debugf("applying forced change: %s", forcedChange)
 
-	// TODO: send the telemetry messages here
-	// afg.applying_forced_authority_set_change
+	efectiveNumberString := strconv.FormatUint(uint64(forcedChange.effectiveNumber()), 10)
+	s.blockState.telemetry.SendMessage(telemetry.NewAfgApplyingForcedAuthoritySetChange(
```

Comment:
@EclesioMeloJunior Is it correct to use the telemetry object from blockState or do I have to create a new one for GrandpaState?

Reply:
Would be better to have a new one for GrandpaState!

Reply:
Done